packages/gestalt/src/Avatar.js

```diff
@@ -16,6 +17,7 @@ const sizes = {
 };
 
 type Props = {|
+  ...SystemProps,
```

Comment:
This won't get picked up by the docgen function. It's not that it gets added with a bad type, it just gets completely ignored.